applications/e-commerce/home.html

```diff
@@ -2,11 +2,26 @@
 <html>
 
 <head>
-
+ <H5>Top 10 colors</H5><hr>
 </head>
 
 <body>
-    Hi class! almost done with git
-    
+    <ol>
+        <li>orange</li>
+        <li>purple</li>
+        <li>black</li>
+        <li>grey</li>
+        <li>blue</li>
+        <li>red</li>
+        <li>yellow</li>
+        <li>white</li>
+        <li>brown</li>
+        <li>Green</li>
```

Comment:
You're seriously going to tell me that green is the worst colour? Worse than yellow??